skxray/io/avizo_io.py

```diff
@@ -263,7 +263,7 @@ def _create_md_dict(clean_header):
     return md_dict
 
 
-def load_amiramesh(file_path):
+def read_amiraMesh(file_path):
```

Comment:
I believe you're referencing this commit, which does not appear to be using `load_amiramesh`

https://github.com/strawlab/scikit-xray/commit/e5ce3d4de2637b839fb6abbf9cf55f989dd92782